Vagrantfile

```diff
@@ -1,37 +1,122 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
-require 'yaml'
+require "yaml"
 
-def load_security
-  fname = File.join(File.dirname(__FILE__), "security.yml")
-  if !File.exist? fname
-    $stderr.puts "security.yml not found - please run `./security-setup` and try again."
-    exit 1
-  end
+# Number of VMs to create
+WORKERS=1
```

Comment:
It would have to be an optional config file with sensible details, so users can still just `vagrant up`, but yeah, absolutely!